test/spawn.jl

```diff
@@ -23,8 +23,8 @@ yes = `perl -le 'while (1) {print STDOUT "y"}'`
 @test length(spawn(pipeline(`echo hello`, `sort`)).processes) == 2
 
 out = readall(`echo hello` & `echo world`)
-@test search(out,"world") != (0,0)
-@test search(out,"hello") != (0,0)
+@test search(out,"world") != 0:-1
+@test search(out,"hello") != 0:-1
```

Comment:
i often want to try to add an inverse test like `@test search(out, "fail") == 0:-1` to help catch these sorts of errors. it's not often easy to write though.